prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/NH/NH-observation-fields.yml

```diff
@@ -0,0 +1,7 @@
+constants:
+  OBX: '/PATIENT_RESULT/ORDER_OBSERVATION/OBSERVATION(%{resultIndex})/OBX'
+
+elements:
+  - name: nh-unwanted-fields-obx
```

Comment:
Sorry I didn't catch this before, but the reliable way to do this is:

  - name: obx-equipment-instance-identifier
    condition: false

It just overrides the entire mapping to OBX-18